Sources/Email/VaporEmail.swift

```diff
@@ -0,0 +1,70 @@
+import Vapor
+
+public struct VaporEmail: Content {
```

Comment:
What about `EmailMessage` here? The Vapor prefix should be reserved for things in the `Vapor` module.